usecases/schema/handler.go

```diff
@@ -173,3 +173,23 @@ func (h *Handler) ShardsStatus(ctx context.Context,
 
 	return h.metaReader.GetShardsStatus(class)
 }
+
+// JoinNode adds the given node to the cluster.
+// Node needs to reachable
+func (h *Handler) JoinNode(ctx context.Context, node string, voter bool) error {
+	nodeAddr, ok := h.clusterState.NodeHostname(node)
```

Comment:
We need to get the raft address. 
That is `raftAddress = nodeAddr.Host() + specific RAFT port`

Reply:
👍 You are right i forgot the port part